### PR TITLE
BaSP-MR-72: Move the logic of the form to the new route

### PR DESCRIPTION
**Pull request by marcohzo**
Split the resource in two screens (Form and Table)
When I press the Add/Edit action button in the table it must redirect to the Form (Use React-Router hooks)
When I finished to add or edit a Super Admin it must redirect me back to the table (Use React-Router hooks)

### CODE REVIEW

**Comment by marcohzo** [comment]
[[Subscriptions] - Move the logic of the form to the new route](https://trello.com/c/P0rLPGSQ/72-subscriptions-move-the-logic-of-the-form-to-the-new-route)

**Review by ghost — APPROVED** [review]
arrive in time to witness the successful live test
resolves conflicts with advice to wait for approval from another partner

**Review by Federico-Peralta — APPROVED** [review]
Tested and works fine!

File: src/Components/Subscriptions/Form/index.jsx
Comment (ale-schmidt) [inline src/Components/Subscriptions/Form/index.jsx:1]: checked and looks ok for me!

**Comment by araefedg** [comment]
The flow works correctly, good job Marco!

**Pull request merged**
